compiler/noirc_evaluator/src/brillig/brillig_ir/procedures/vector_pop_front.rs

```diff
@@ -91,11 +91,6 @@ pub(super) fn compile_vector_pop_front_procedure<F: AcirField + DebugToString>(
                 source_capacity.address,
                 BrilligBinaryOp::Sub,
             );
-            brillig_context.codegen_initialize_vector_metadata(
-                target_vector,
-                target_size,
-                Some(source_capacity),
-            );
```

Comment:
This seems to fix the odd bit size errors while not failing any tests. @vezenovm sanity check for this fix. It is odd to just remove code, at the same time though the rc is one so we are directly mutating the same slice.